sources/de-bkg.json

```diff
@@ -3,7 +3,7 @@
     "edtf:inception": "2019-05-10",
     "fullname": "Bundesamt fur Kartographie und Geodasie",
     "id": 1377649075,
-    "key": "",
+    "key": "id",
```

Comment:
I see extra carriage returns in this PR, except on this json file. Probably not a blocker, but calling out the inconsistency in case this needs any extra adjustment..?

Reply:
The concordance key postfix is `id`, so I filled this missing bit in, now that we're using it for concordances, too. 

We said before we were using it for concordances, but we hadn't filled in this bit.